app/google-services.json

```diff
@@ -0,0 +1,69 @@
+{
+  "project_info": {
+    "project_number": "735289696982",
+    "firebase_url": "https://bcycle-6d8e7.firebaseio.com",
+    "project_id": "bcycle-6d8e7",
+    "storage_bucket": "bcycle-6d8e7.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:735289696982:android:5507ff93ef22fdece95f77",
+        "android_client_info": {
+          "package_name": "pl.bcycle.firebasetest"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "735289696982-7rh31vd7q8tfpreum3fma5kddqq3mrqs.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyDEnptQvlMZ9f9w-zHJoey6EYGp26EvN8U"
```

Comment:
API keys should not be stored in a public repo.

Reply:
That's official google's approach

Reply:
Its not that confidential like the admin credentials was. It also can be easily extracted from the apk like showed here: https://wbrawner.com/2019/08/01/is-it-safe-to-store-your-google-services-json-in-a-public-git-repo/. 
You can find more discussions about that file here (also see linked threads): https://stackoverflow.com/questions/37358340/should-i-add-the-google-services-json-from-firebase-to-my-repository

To be clear, if we were doing some commercial project i would think twice about storing it in git, but since its only for university i think its ok to leave it as it is.